ports/espressif/boards/makerfabs_tft7/mpconfigboard.mk

```diff
@@ -15,6 +15,8 @@ CIRCUITPY_ESP_PSRAM_FREQ = 80m
 
 CIRCUITPY_DOTCLOCKFRAMEBUFFER = 1
 
+CIRCUITPY_SDIOIO = 1
```

Comment:
I agree with @bill88t that this should be in mpconfigport.mk as default on for chips that support it.

Reply:
Thanks, done.